src/index.js

```diff
@@ -1,29 +1,56 @@
 import puppeteer from 'puppeteer';
+import readline from 'readline';
 import db from './utils/db';
 import ticketManager from './utils/tickets';
 
 const main = async () => {
+  console.log(`Starting service... <q! to exit>`);
+  var should_break = false;
```

Comment:
`var` is a dirty word in JS these days, [here's a good but long post](https://hackernoon.com/why-you-shouldnt-use-var-anymore-f109a58b9b70) - tl;dr - use `let` for things that need to be re-assigned, and `const` otherwise. This one's a `let`. If you copy and paste old code you'll see it quite a bit (that was the standard for... 10 years?) but many modern linters will complain about its use.

Reply:
Roger roger